src/mpyl/steps/steps.py

```diff
@@ -16,7 +16,9 @@
 
 
 class Steps:
+    """ Executes the pipeline with its build properties """
     _step_executors: set[Step]
+    """ A set of all the steps inside the pipeline """
```

Comment:
Private members don't need to be documented

Reply:
Removed comment